packages/gatsby-theme-carbon/gatsby-node.js

```diff
@@ -1,6 +1,7 @@
 const path = require('path');
 const fs = require('fs');
 const mkdirp = require('mkdirp');
+const startCase = require('lodash.startcase');
```

Comment:
Possibly controversial? `gatsby-plugin-mdx` includes the whole package already. I don't think this will be bundled to the browser.

Reply:
Yeah, shouldn't be a problem